src/main/java/rs/teslaris/core/controller/ResearchAreaController.java

```diff
@@ -28,9 +25,8 @@ public class ResearchAreaController {
     private final ResearchAreaService researchAreaService;
 
     @GetMapping
-    public Page<ResearchAreaResponseDTO> getResearchAreas(Pageable pageable) {
-        return researchAreaService.findAll(pageable).map(
-            ResearchAreaConverter::toResponseDTO);
+    public List<rs.teslaris.core.dto.commontypes.ResearchAreaDTO> getResearchAreas() {
+        return researchAreaService.getResearchAreas();
```

Comment:
Why this is the part of this PR? It is not connected to upgrade of spring boot, right?